### PR TITLE
Adding static tag for remote CDN compatibility

### PR DESCRIPTION
**Pull request by ryanbagwell**

### CODE REVIEW

**Comment by mkoistinen** [comment]
LGTM. @FinalAngel?

File: djangocms_admin_style/templates/admin/inc/branding.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {% load i18n %}
+{% load static from staticfiles %}
```
Comment (FinalAngel) [inline djangocms_admin_style/templates/admin/inc/branding.html]: shouldn't it be just ``{% load staticfiles %}``?

File: djangocms_admin_style/templates/admin/inc/branding.html [hunk 2]
```diff
@@ -27,6 +28,6 @@
     </a>
 </div>
 
-<script src="/static/js/drag-touch-support.js"></script>
-<script src="/static/js/base-admin.js"></script>
-<script src="/static/js/jquery.ui.touch-punch.min.js"></script>
+<script src="{% static "js/drag-touch-support.js" %}"></script>
```
Comment (FinalAngel) [inline djangocms_admin_style/templates/admin/inc/branding.html]: use single quotes inside src=""

**Comment by yakky** [comment]
LGTM

File: djangocms_admin_style/templates/admin/inc/branding.html (hunk 1 above)
Comment (mkoistinen) [inline djangocms_admin_style/templates/admin/inc/branding.html]: This is fine too.

**Comment by ryanbagwell** [comment]
Thank you!

**Comment by mkoistinen** [comment]
Sorry, was committed pre-maturely.

**Comment by mkoistinen** [comment]
@ryanbagwell can you change the double-quotes to single-quotes as per our standards? (sorry, seems like a small thing, but would be helpful).

**Comment by ryanbagwell** [comment]
no problem. See #231

**Pull request merged**